cmd/mt-store-cat/full.go

```diff
@@ -9,7 +10,7 @@ import (
 	"github.com/grafana/metrictank/mdata"
 )
 
-func chunkSummary(store *mdata.CassandraStore, tables []string, metrics []Metric, keyspace, groupTTL string) error {
+func chunkSummary(ctx context.Context, store *mdata.CassandraStore, tables []string, metrics []Metric, keyspace, groupTTL string) error {
```

Comment:
that `ctx` doesn't seem to be used

Reply:
for consistency, all these functions that go out and collect and print data get a context arg